hw/ip/csrng/data/csrng_testplan.hjson

```diff
@@ -69,9 +65,6 @@
       tests: ["csrng_err"]
     }
     {
-      //TODO For the following untested features add it to the test or create explicit coverpoints in csrng_cmd_cg:
-      //TODO - Verification of otp_en_csrng_sw_app_read is currently untested.
-      //TODO - FIPS bit going low from entropy source is currently untested.
```

Comment:
Have you checked whether this is already tested? My understanding is that this PR "just" adds the missing coverpoints. In case the features aren't being tested yet, this will show in the corresponding coverpoints not being hit. Right?

Reply:
Specifically I am bit worried about the FIPS bit. Because the now added coverpoint will tell us if tests read both output data generated from FIPS compliant and in-compliant seeds. But it won't show us whether after reading FIPS compliant bits once, FIPS in-compliant bits can be read again. I.e. whether the FIPS bit going low at the input eventually becomes visible at the output again. This is pretty important from a security perspective. Have you insight into whether is tested somehow?

Reply:
> Have you checked whether this is already tested? My understanding is that this PR "just" adds the missing coverpoints. In case the features aren't being tested yet, this will show in the corresponding coverpoints not being hit. Right?

Yes, this change only adds coverpoints.

Reply:
In the interest of time, let me create an issue for that. Then we can move ahead and proceed with this PR.

Reply:
So in terms of toggle coverage, FIPS bit going low from entropy source is covered:
![image](https://github.com/lowRISC/opentitan/assets/34654485/942ebffa-d780-4743-b099-84a0a0dd8cd0)

I am not aware of any test that tests the proper behavior here though: i.e. when entropy source provides a non-FIPS value the value that CSRNG outputs is also non-FIPS.